packages/library/src/providers/ModalProvider/ModalProvider.tsx

```diff
@@ -19,6 +19,7 @@ import { useBreakpoint } from '@deriv/quill-design';
 type TModalState = {
     // Add your modal states here
     modalType?: string; // Just an example
+    selectedJurisdiction?: string;
```

Comment:
dont use modalstate for this 
we have setCfdState for cfd related stuff

Reply:
we have a cfd state for cfd modal?